docs/networking.md

```diff
@@ -0,0 +1,68 @@
+# Networking
+
+The default networking mode is `docker-bridge`, which means that the default docker bridge will be used for the networking setup.
+The default docker bridge is a local `docker0` interface, giving out local addresses to containers in the `172.17.0.1/16` range.
+
+Ignite also supports integration with [CNI](https://github.com/containernetworking/cni), the standard networking interface
+for Kubernetes and many other cloud-native projects and container runtimes. Note that CNI itself is only an interface, not
+an implementation, so if you use this mode you need to install an implementation of this interface. Any implementation that works
+with Kubernetes should technically work with Ignite.
+
+## Comparison
+
+### docker-bridge
+
+**Pros:**
+
+- **Quick start**: If you're running docker, you can get up and running without installing extra software
+- **Port mapping support**: This mode supports port mappings from the VM to the host
+
+**Cons:**
+
+- **docker-dependent**: By design, this mode is can only be used with docker, and is hence not portable across container runtimes.
+- **No multi-node support**: The IP is local (in the `172.17.0.0/16` range), and hence other computers can't connect to your VM's IP address.
```

Comment:
This IP range differs from the one given above, I believe this one here (`172.17.0.0/16`) is correct and the one above should be changed.